image_proc/launch/image_proc.launch

```diff
@@ -3,7 +3,12 @@
 
   <arg name="manager" /> <!-- Must be globally qualified -->
   <arg name="respawn" default="false" />
-  <!-- TODO Arguments for debayer, interpolation methods? -->
+  <arg name="approximate_sync" default="false" />
+  <arg name="debayer_method" default="0" doc="Debayering algorithm:
```

Comment:
Hi,
I don't see where this is used in the pipeline (or even in this launch file). Can you clarify what you're trying to expose?